src/main/java/duke/Task.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * superclass of other task classes
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task (String description) {
+        this.description = description;
+        this.isDone = false;
+
+    }
+
+    /**
+     * determine if a task is done or undone
+     * @return done or undone
+     */
+    public String getStatusIcon() {
+        return this.isDone ? "X" : " ";
+    }
+
+    /**
+     * mark a task as done
+     * @return a message to verify that task is marked as done
+     */
+    public String markedDone() {
+        this.isDone = true;
+        return  this.message();
```

Comment:
Did you forget to remove the additional space? Noted it in other places as well.